Article/Article.js

```diff
@@ -3,18 +3,25 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = domElement.querySelector(".expandButton");
+    this.collapseButton = domElement.querySelector(".collapseButton");
+    this.buttonsGroup = domElement.querySelector(".article-buttons")
     // Using your expandButton reference, update the text on your expandButton to say "expand"
+    this.expandButton.textContent = "expand";
+    this.collapseButton.textContent = "collapse";
     
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.buttonsGroup.addEventListener("click", this.expandArticle.bind(this));
   }
 
   expandArticle() {
```

Comment:
Nice work getting the close button to work, you can also remove lines 9, 10, 13, 22, 23 and use this logic instead:

```
  expandArticle() {
    // Using our reference to the domElement, toggle a class to expand or hide the article.
    this.domElement.classList.toggle("article--open");
    
    this.expandButton.textContent === "Expand"
      ? (this.expandButton.textContent = "Collapse")
      : (this.expandButton.textContent = "Expand");
  }
```